script.js

```diff
@@ -185,8 +185,25 @@ $(document).ready(function() {
   }
   
   function setWins(wins) {
-      $('#win-count').text(wins);
-      localStorage.setItem('wins', wins);
+    let result = ''
+
+    if(wins > 0) {
+      result = wins
+    }
+
+    if(wins > 10) {
+      const extraWins = wins - 10;
+      result = `10 +  ${extraWins}`
+    }
+
+    if(wins >= 20) {
+      const roundNumber = Number(wins.toString().slice(0, 1))
```

Comment:
Try 100 wins, and this will output '10 x 1 + 90', which is incorrect (should be 10 x 10).
Use this formula: Math.round(Math.floor(wins / 10))